archivebox/core/urls.py

```diff
@@ -31,4 +31,5 @@
     path('index.html', RedirectView.as_view(url='/')),
     path('index.json', static.serve, {'document_root': settings.OUTPUT_DIR, 'path': 'index.json'}),
     path('', MainIndex.as_view(), name='Home'),
+    path('public/', PublicArchiveView.as_view())
```

Comment:
why a separate URL, cant this just be the default homepage when PUBLIC_INDEX=True?

Reply:
Would you like this view to replace the OldHome then?